packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -286,6 +286,11 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
       const result = buildCss(interpolation, updatedMeta);
       variables.push(...result.variables);
       css.push(...result.css);
+
```

Comment:
This is all we need for both issues.
At first I was having a different logic (https://github.com/atlassian-labs/compiled/pull/805/commits/65633a0f0c25802299a1a3821549b90cbc774454), then realised that could be easily simplified in here :)

Reply:
Yep, this is much better :)

Reply:
Yep, this is much better! :)